docs/css/moose.css

```diff
@@ -25,6 +25,14 @@
 }
 /* End Collapse Nav-bar correctly */
 
+/* Get rid of the Previous and Next buttons */
+ul.nav.navbar-right > li:nth-of-type(2), ul.nav.navbar-right > li:nth-of-type(3){
+    display: none;
+}
+/* Get rid of the Previous and Next buttons */
+
+/* Start 2nd level deep and recursively dive into any sub ul/li */
```

Comment:
I delete that line. That is remnants of trying to get the menu system to work for us...